crates/miden-lib/asm/kernels/transaction/lib/prologue.masm

```diff
@@ -496,24 +496,31 @@ proc.process_account_data
     exec.memory::set_init_nonce
     # => [ACCOUNT_COMMITMENT]
 
-    # validates and stores account storage slots in memory.
+    # validates and stores account storage slots in memory
+    # account storage commitment is also stored as an initial one in the global inputs
     exec.memory::get_acct_storage_commitment
+    exec.memory::set_init_account_storage_commitment
     exec.account::save_account_storage_data
     # => [ACCOUNT_COMMITMENT]
 
     # set the new account code commitment to the initial account code root
     # this is used for managing code commitment updates
+    # account code commitment is also stored as an initial one in the global inputs
     exec.memory::get_acct_code_commitment
+    exec.memory::set_init_account_code_commitment
```

Comment:
Is this call actually necessary? If I understand things correctly, `get_acct_code_commitment` reads data from memory location `8204` and then `set_init_account_code_commitment` writes the same data into location `8204`.

If so, we can probably get rid of `memory::set_init_account_code_commitment` procedure.